### PR TITLE
Created new contact UI

### LINKED ISSUE #6: Create a new contact UI

### PR DESCRIPTION
**Pull request by anandhakrishnanaji**
Fixes #6

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Toastr.success("Contact deleted successfully.");
+    onClose();
+  };
+
+  return (
+    <Alert
+      isOpen
+      size="lg"
+      title="Delete contact?"
+      message="Are you sure you want to continue with the deletion of the contact? This cannot be undone.
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx]:
```suggestion
      message="Are you sure you want to continue with deleting this contact? This cannot be undone.
```

File: app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Form";
+
+import { CONTACTS_FORM_INITIAL_FORM_VALUES } from "../constants";
+
+const NewContactPane = ({ showPane, setShowPane }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx]: https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663590973475389?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+              style="primary"
+              type="submit"
+              onClick={() => setSubmitted(true)}
+            />
+            <Button
+              label="Cancel"
+              size="large"
+              style="text"
+              onClick={onClose}
+            />
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:85]: Go through the Slack thread.

**Review by yedhink — CHANGES_REQUESTED** [review]
@anandhakrishnanaji

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx (hunk 3 above)
Comment (anandhakrishnanaji) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:85]: Did you mean to set the type prop as "reset"?
Shouldn't the cancel button be used to hide the pane? in the examples in the neetoui docs, the cancel button hides the pane.
Or should I rename the button as "Reset"

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:85]: @anandhakrishnanaji The cancel hides the pane and in turn resets the data. From formik perspective this is a button that can reset the data. Thus it's better to be explicit here and set the type as `reset`.

**Pull request merged**